assets/common.sh

```diff
@@ -63,7 +63,7 @@ setup_kubectl() {
     fi
     exe kubectl config set-cluster "$CLUSTER_NAME" "${set_cluster_opts[@]}"
 
-    exe kubectl config set-context "$CONTEXT_NAME" --cluster="$CLUSTER_NAME"
+    exe kubectl config set-context "$CONTEXT_NAME" --cluster="$CLUSTER_NAME" --user "$AUTH_NAME"
```

Comment:
With this fix the generated kubectl config file looks like this, when executed with a build of a container that has this code everything works as expected.
```
apiVersion: v1
clusters:
- cluster:
    certificate-authority-data: 
xxxxxxx
    server: https://xxxxx.us-west-2.eks.amazonaws.com
  name: cluster
contexts:
- context:
    cluster: cluster
    namespace: default
    user: auth
  name: kubernetes-resource
current-context: kubernetes-resource
kind: Config
preferences: {}
users:
- name: auth
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1alpha1
      args:
      - token
      - -i
      - eks-o1
      command: aws-iam-authenticator
      env: null
```